ext/opentelemetry-ext-opentracing-shim/src/opentelemetry/ext/opentracing_shim/__init__.py

```diff
@@ -327,6 +327,7 @@ class ScopeShim(opentracing.Scope):
     def __init__(self, manager, span, span_cm=None):
         super().__init__(manager, span)
         self._span_cm = span_cm
+        attach(set_value("scope_shim", self))
```

Comment:
is it worth detaching from this context when the scope shim is closed?

Reply:
Excellent point, fixed.